utils/middlewares/service.ts

```diff
@@ -0,0 +1,31 @@
+import { Service as ApiService } from "io-functions-commons/dist/generated/definitions/Service";
+import { IRequestMiddleware } from "io-functions-commons/dist/src/utils/request_middleware";
+import { ResponseErrorFromValidationErrors } from "italia-ts-commons/lib/responses";
+
+import { Logo as ApiLogo } from "../../generated/definitions/Logo";
+
+/**
+ * A middleware that extracts a Service payload from a request.
+ */
+export const ServicePayloadMiddleware: IRequestMiddleware<
+  "IResponseErrorValidation",
+  ApiService
+> = request =>
+  Promise.resolve(
+    ApiService.decode(request.body).mapLeft(
```

Comment:
You are decoding for a Service but you need a ServicePayload. Am i wrong?

Reply:
Fixed in [9bbe257](https://github.com/pagopa/io-functions-services/pull/68/commits/9bbe257016950fbe4605124203d3f558fd5f8b72)